scripts/http-vuln-cve2014-3704.nse

```diff
@@ -216,9 +216,14 @@ local function do_sql_query(host, port, uri, user)
     }
   }
   local res = http.post(host, port, uri .. "/user/login", opt, nil, r)
-  --TODO: Check return status
 
-  return user, passwd
+  if string.match( res.body, 'mb_strlen() expects parameter 1 to be string' ) then
```

Comment:
A better check has been written in the re-written exploit, [PR #905](https://github.com/rewanth1997/nmap/blob/2f541cc74123e95474f74b65a8f39d817c2177f6/scripts/http-vuln-cve2014-3704.nse#L379)